bin/cpip

```diff
@@ -96,12 +96,12 @@ set_logging "$default_tag"
 
 # check os to find the right compilers
 if [[ $OSTYPE == "linux-gnu" ]]; then
-    cc_pkg="gcc_linux-64"
-    cxx_pkg="gxx_linux-64"
-    gfortran_pkg="gfortran_linux-64"
-    ld_impl="ld_impl_linux-64"
-    swig_pkg="swig"
-    pkg_config="pkg-config"
+    cc_pkg="gcc_linux-64==7.2.0"
```

Comment:
Would be nice if we could specify the compiler version in an env variable, but we can revisit that later.

Reply:
Ticket to tackle this: #17